src/coreclr/vm/frames.h

```diff
@@ -860,13 +860,24 @@ class RedirectedThreadFrame : public ResumableFrame
 #elif defined(TARGET_ARM64)
             Object** firstIntReg = (Object**)&this->GetContext()->X0;
             Object** lastIntReg  = (Object**)&this->GetContext()->X28;
+#elif defined(TARGET_LOONGARCH64)
+            Object** firstIntReg = (Object**)&this->GetContext()->Tp;
+            Object** lastIntReg  = (Object**)&this->GetContext()->X0;
 #else
             _ASSERTE(!"nyi for platform");
 #endif
             for (Object** ppObj = firstIntReg; ppObj <= lastIntReg; ppObj++)
             {
                 fn(ppObj, sc, GC_CALL_INTERIOR | GC_CALL_PINNED);
             }
+#if defined(TARGET_LOONGARCH64)
+            firstIntReg = (Object**)&this->GetContext()->S0;
+            lastIntReg  = (Object**)&this->GetContext()->S8;
+            for (Object** ppObj = firstIntReg; ppObj <= lastIntReg; ppObj++)
+            {
+                fn(ppObj, sc, GC_CALL_INTERIOR | GC_CALL_PINNED);
+            }
+#endif
```

Comment:
```suggestion
```
It is ok to just include fp in the conservative scanning to keep things simple. Conservative stack scanning is not high performance config.

Reply:
Ok, thanks.